google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -498,29 +498,21 @@ private static Severity severityFor(Level level) {
     if (level instanceof LoggingLevel) {
       return ((LoggingLevel) level).getSeverity();
     }
-
-    switch (level.intValue()) {
-        // FINEST
-        // FINER
-        // FINE
-      case 300:
-      case 400:
-      case 500:
-        return Severity.DEBUG;
-        // CONFIG
-        // INFO
-      case 700:
-      case 800:
-        return Severity.INFO;
-        // WARNING
-      case 900:
-        return Severity.WARNING;
-        // SEVERE
-      case 1000:
-        return Severity.ERROR;
-      default:
-        return Severity.DEFAULT;
+    // Choose the severity based on Level range.
+    // The assumption is that Level values below maintain same numeric value
+    int value = level.intValue();
```

Comment:
Thanks :-)
As for `level == null` - I can definitely add it, but it will make this code to be null tolerant which is in turn could have an impact on other functions like `logEntryFor()`. I guess I will leave it as is